### PR TITLE
Update config.json template

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

**Review by Amir-61 — CHANGES_REQUESTED** [review]
@loay Thanks for the PR. Please add a test verifying your changes.

CC: @bajtos

**Comment by loay** [comment]
There is no test needed. the changes are in the example tutorial not the actual component
it is an issue with the model .. not the component itself

File: example-3.0/server/config.json [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
   "restApiRoot": "/api",
   "host": "0.0.0.0",
   "port": 3000,
-  "url": "http://localhost:3000/",
   "remoting": {
-    "cors": false
-  }
+    "context": {
+      "enableHttpContext": false
+    },
```
Comment (Amir-61) [inline example-3.0/server/config.json:8]: HTTPContext is removed in LB 3.0.  We should have 
```
"context": false
```

File: example-2.0/server/model-config.json [hunk 2]
```diff
@@ -26,6 +26,6 @@
   },
   "container": {
     "dataSource": "storage",
-    "public": true
+    "public": false
```
Comment (Amir-61) [inline example-2.0/server/model-config.json]: I don't think if we should hide `contianer` model in explorer. @bajtos @superkhau thought?

**Review by Amir-61 — CHANGES_REQUESTED** [review]
Reviewed.

File: example-3.0/server/config.json (hunk 1 above)
Comment (loay) [reply example-3.0/server/config.json:8]: Here is the config template:
https://github.com/strongloop/loopback-workspace/blob/master/server/config.json

File: example-2.0/server/model-config.json (hunk 2 above)
Comment (loay) [reply example-2.0/server/model-config.json]: I am ok with that for example 3, but for 2, it is a must. Another model could be created to state the files downloaded or uploaded but it shouldn't be pointing to the container

**Comment by bajtos** [comment]
@loay @Amir-61 IMO, we need to fix this problem in loopback-component-storage and its filesystem provider. Start here: https://github.com/strongloop/loopback-component-storage/blob/080f18a985672f9495b69c7e082b38a0e3ffcc1f/lib/providers/filesystem/index.js#L153-L154

I think the example app is correct, the `container` model has to be exposed - it's the API this example is demonstrating. cc @superkhau

**Comment by loay** [comment]
@bajtos that's fine, I left the container exposed, however the current template in `config.json` is wrong and it had to be updated with the proper template for loopback 3.

**Comment by bajtos** [comment]
> however the current template in config.json is wrong and it had to be updated with the proper template for loopback 3.

Makes sense. IMO, the project layout should be fixed in a standalone pull request, not as a security fix.

**Comment by loay** [comment]
@Amir-61  PTAL 
thanks

**Review by Amir-61 — APPROVED** [review]

**Pull request merged**